nilearn/reporting/_get_clusters_table.py

```diff
@@ -327,27 +336,30 @@ def get_clusters_table(stat_img, stat_threshold, cluster_threshold=None,
         # If the stat threshold is too high simply return an empty dataframe
         if np.sum(binarized) == 0:
             warnings.warn(
-                'Attention: No clusters with stat {0} than {1}'.format(
-                    'higher' if sign == 1 else 'lower',
+                "Attention: No clusters with stat {0} than {1}".format(
+                    "higher" if sign == 1 else "lower",
                     stat_threshold * sign,
                 )
             )
             continue
 
         # Now re-label and create table
         label_map = label(binarized, bin_struct)[0]
-        # Save label maps as nifti objects
-        label_maps.append(new_img_like(stat_img, label_map, affine=affine))
         clust_ids = sorted(list(np.unique(label_map)[1:]))
         peak_vals = np.array(
-            [np.max(temp_stat_map * (label_map == c)) for c in clust_ids])
+            [np.max(temp_stat_map * (label_map == c)) for c in clust_ids]
+        )
         # Sort by descending max value
         clust_ids = [clust_ids[c] for c in (-peak_vals).argsort()]
 
+        re_label_map = np.zeros_like(label_map)
```

Comment:
To make sure I understand: the relabeling is meant to keep consistency after the sorting operation ?
If yes, this could be added explicitly.